raiden/api/rest.py

```diff
@@ -877,6 +883,23 @@ def leave(self, registry_address: typing.PaymentNetworkID, token_address: typing
         ]
         return api_response(result=closed_channels)
 
+    @light_client_only
```

Comment:
Awesome decorator, really good new feature

Reply:
Thankkkk you!

Reply:
it's very nice that you have added this decorator, but i'm not so sure about the name. i would maybe reference the logic of the function in its name, with a name like `@requires_api_key` or something along those lines.

not a blocker though

Reply:
Agree with @mortelli . some ideas: 

@registered_light_client 
@authorized_light_client
@light_client_auth_required